pkgs/applications/version-management/git-and-tools/pre-commit/default.nix

```
@@ -1,11 +1,11 @@
 { stdenv, python3Packages }:
 with python3Packages; buildPythonApplication rec {
   pname = "pre_commit";
-  version = "1.11.1";
+  version = "1.13.0";
 
   src = fetchPypi {
     inherit pname version;
```

```suggestion
    pname = "pre_commit";
    inherit version;
```

Because `pname` in the url is formatted like this.